frontend/integration-tests/tests/metalkube/maintenance.scenario.ts

```diff
@@ -0,0 +1,39 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+
+import { appHost } from '../../protractor.conf';
+import * as crudView from '../../views/crud.view';
+import * as bareMetalHostView from '../../views/metalkube/baremetalhost.view';
+
+describe('Start and stop maintenance master-0', () => {
+  it('go to /k8s/all-namespaces/baremetalhosts', async() => {
+    // FIXME: prefer navigating using the actual nav pane, to test that it's working and we're not just bypassing it
+    await browser.get(`${appHost}/k8s/all-namespaces/baremetalhosts`);
+    await crudView.isLoaded();
+    // confirm that we're on the right page by testing that the "create" button adds a host
+    expect(crudView.createYAMLButton.getText()).toEqual('Add Host');
+  });
+
+  it('find master-0 and check its status', async() => {
+    browser.wait(until.presenceOf(crudView.rowForName('openshift-master-0')));
```

Comment:
can you make variable from `openshift-master-0` please?

Reply:
will do.